### DISCUSSION TITLE
Password reset

### DISCUSSION THREAD

**Discussion by quer0016** [JanssenProject/jans#8740] [Q&A]
Hello, team Is there a possibility in flex to have something to reset the password in case of forgetting? For example, there is a Link synchronization with an external AD/LDAP and can it be done?

**Comment by nynymike** [comment]
This is basicaly just email authn. You could use on of these flows: https://github.com/GluuFederation/agama-smtp 

If successful, to reset an LDAP PW you'd have to find the DN of the subject, and then do an LDAP MODIFY with the appropriate AD LDAP credentials on the respective entry.
